tv_apps/fling-player/js/fling_player.js

```diff
@@ -413,9 +450,23 @@
   proto.onLoadRequest = function (e) {
     this.resetUI();
     this.showLoading(true);
-    this.toggleInitialMessage(true);
     this._player.load(e.url);
     this.play();
+    this._connector.getControllingDeviceInfo()
+        .then((info) => {
```

Comment:
Still wrong indentation.

`this._connector.getControllingDeviceInfo().then(info => {
  var msg = ...
  var name = ...
`
Isn't that plenty?